src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1397,6 +1397,7 @@ MavlinkReceiver::handle_message_set_attitude_target(mavlink_message_t *msg)
 						case MAV_TYPE_OCTOROTOR:
 						case MAV_TYPE_TRICOPTER:
 						case MAV_TYPE_HELICOPTER:
+						case MAV_TYPE_VTOL_RESERVED2:
 							att_sp.thrust_body[2] = -set_attitude_target.thrust;
```

Comment:
This will work only in MC mode, `att_sp.thrust_body[0]` should be used when in fixed mode.

Reply:
Tried to check that:
https://github.com/PX4/Firmware/pull/13141/commits/db987e90f6134ec38d524104c3923f83312fad1e

(Due to [this](https://github.com/PX4/Firmware/blob/f885d2274a563278b5ca501e4c19365ad0cd790d/src/modules/vtol_att_control/vtol_att_control_main.cpp#L387) `switch`)

Reply:
Now checked and filling in true `thrust_body` field : https://github.com/PX4/Firmware/pull/13141/files